src/polyfem/state/StateInit.cpp

```diff
@@ -203,6 +216,15 @@ namespace polyfem
 
 		this->args = jse.inject_defaults(args_in, rules);
 
+		std::string out_path_log = this->args["output"]["log"]["path"];
```

Comment:
@zfergus this line doesn't work with strings, do you know why?

Reply:
no, this looks like it would work